unit_tests/testGenReqTables/subtest01/testA.inp

```diff
@@ -0,0 +1,8 @@
+!> @beginreq
+!> - MPACT shall run on Mars.
```

Comment:
While I did have a good laugh at this, I think it would be good to update these to be correct w.r.t. the ticket and what the actual test is

Reply:
Okay. So would you like some actual test inputs/requirements (e.g., from MPACT regression tests) to fill in those testA, B C instead of the pseudo requirement/test? I thought this is open source and not has to be tied to reactor application, so I tried not to put the CASL test/requirement in.

Reply:
I think the only thing would be the ticket number, and I don't see why that would be an issue.

The rest of the requirements are actually that the open source requirements parsing tool (not MPACT) do the appropriate thing.

Does that clarify things for you?

Reply:
oh, so just put the requirement statement and ticket number associated with a realistic test?

Reply:
Yeah, so here the requirements statement might be:
```
!> @beginreq
!> - The GenerateRequirements.py script shall be able to parse requirement
!>   descriptions in this format.
!> - Ticket 5690
!> @endreq
```

Reply:
ahhhhhhhhhhhhhhhhhh, finally got it!!

Reply:
All tests updated to show what's being tested.